tools/travis/travis_linux_script.sh

```diff
@@ -395,6 +395,16 @@ if [ $? -ne 0 ] ; then exit 1 ; fi
 cd .. && rm build -rf
 echo -en 'travis_fold:end:script.build.unit_test_ns0_full\\r'
 
+echo -e "\r\n== clang-format & Static Code Analyzer ==" && echo -en 'travis_fold:start:script.build.clang_format_static_analyzer\\r'
```

Comment:
I would put this into the analyze section inside CLang.
This should only be run once for every commit.

https://github.com/open62541/open62541/blob/0b5b11e2f3003ce8e62e9b6237ebb0e444606e7f/tools/travis/travis_linux_script.sh#L102

Reply:
In fact, we can remove this section after a clang-format is done on the whole code base and the fix commits are merged.
I just included it as quality criteria because clang-format or one of the SCAs broke the build.
So this section guarantees, that the project is still building after performing a clang-format on the whole code base, which is not included within this PR.

Reply:
clang-format will never be completely done for the whole code base. The only thing we could do is to gradually enforce some specific format for the changed lines of a specific PR. Otherwise it leads to many merge conflicts.
E.g. something like this:
https://github.com/material-foundation/clang-format-ci


So express it in other words:
After merging this PR, the whole code base conforms to the REDUCED setting, correct? And `make clangformat` checks that for every new commit?
Shouldn't this be checked with clang-tidy? I'm a bit confused here...